<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatServerDialog</class>
 <widget class="QDialog" name="ChatServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatServerDialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PORT:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8080</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Create">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CreateServer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Close">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CloseServer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
